### ISSUE TITLE
Gather a list of features and bugs we fixed in 2018.2

### ISSUE THREAD

**Issue by onox** [c172p-team/c172p-detailed#958]
We should gather a list of all the new features, major changes, and bugs we fixed. Please create a list (use \* to create bullet points) here.

* TODO

Do not forget to update `NEWS.md` before creating the release. See https://github.com/c172p-team/c172p-detailed/wiki/Release-process-and-versioning

**Comment by onox** [comment]
@wlbragg @gilbertohasnofb Can you edit the comment above after closing PR #964 and describe what has been fixed? Seems quite a lot :)

**Comment by onox** [comment]
@wlbragg Can I delete branch `regression-to-3.2` and `state-overlay-test`?

**Comment by gilbertohasnofb** [comment]
@wlbragg I added a small list of fixes to the first post, but feel free to describe things in more detail or to add more items.

**Comment by gilbertohasnofb** [comment]
@wlbragg I decided to use your list of fixes from the forum.

**Comment by wlbragg** [comment]
@onox, yes, delete away!

**Comment by gilbertohasnofb** [comment]
@wlbragg I just updated the first post, could you please check if I did not miss anything?

**Comment by wlbragg** [comment]
@gilbertohasnofb 
I added or modified the following...

* Better smoothness of all 3D models including variants
* New landing gear strut steps and strut to fuselage connection
* More realistic looking tires including all variants

Thank you so much for all the hard work on this. You ready to go again?

**Comment by gilbertohasnofb** [comment]
> You ready to go again?

Oh, yeah! Whenever you want to attack the cockpit issues :smile: Or are there any other things left with the external 3D model apart from the UV-map of the wing tips?

**Comment by wlbragg** [comment]
I added
* Cutout holes for wheel in fairings
* New exterior door handles

**Comment by wlbragg** [comment]
>are there any other things left with the external 3D model apart from the UV-map of the wing tips?

I guess it's time to go inside and see if we cant move the panel.

**Comment by gilbertohasnofb** [comment]
> I guess it's time to go inside and see if we cant move the panel.

:+1: :+1: :+1:

**Comment by gilbertohasnofb** [comment]
@wlbragg I added some new stuff to the list in relation to the work done in that last PR, could you please take a look I did not forget anything?

**Comment by wlbragg** [comment]
@gilbertohasnofb 

Added...
    * Nose mesh retopologize
    * Added rudder pedal push rod model
    * Added sump port models
    * Added animated cabin air vent
    * Added engine compartment vent
    * Added wing vent holes
    * Improved bumpspec modeling
    * New cowling handhold

Feel free to reword any of it.

**Comment by gilbertohasnofb** [comment]
I added:

* Improvements to the front glass
* Improved ground equipment

**Comment by gilbertohasnofb** [comment]
@wlbragg Would you help me add stuff from the last PR? I am not sure I know everything you did to the damage model, so perhaps I am not the right person to write these.

**Comment by wlbragg** [comment]
Yeah, I'll see if I can finish the list.

**Comment by wlbragg** [comment]
I added the following...
* Model strobe light bulbs on NAV light casings
* Model rudder NAV light
* Re-factor damage models to match model improvements
* Optimize damage system to use less models
* Model ribs to show through broken wing
* Add two sided prop blur to livery system

**Comment by onox** [comment]
@wlbragg @gilbertohasnofb @stuartbuchanan When is 2018.2 going to be released? In May?

**Comment by onox** [comment]
@c172p-team/developers I've created some 2018.2 milestones. These are for issues that have been fixed. When creating new issues, please connect them to the "Future"/"3D modeling" milestones.

**Comment by gilbertohasnofb** [comment]
@onox According to the wiki:

> Release timeline
> [...]
> 2018.2.1 (due 2018 May 17) 

But I am not sure they are on schedule or not.

**Comment by onox** [comment]
Yeah that's what I thought by looking at the deadlines of some previous milestones. I've set the deadlines for the 2018.2 milestones to the 13th of May (Sunday).

**Comment by onox** [comment]
Added:

- Implemented engine damage when flying at a RPM higher than the red mark

**Comment by legoboyvdlp** [comment]
Please also add

- More advanced oil management and consumption calculation

**Comment by onox** [comment]
@gilbertohasnofb Do we have a list of issues that we really want to fix this weekend?

- #967 (the image in the `README.md` is no longer visible. Could you make a new one with the new green retro livery perhaps?)

**Comment by gilbertohasnofb** [comment]
> Do we have a list of issues that we really want to fix this weekend?

I think all these new cosmetic things @wlbragg and I are working on should not go on this release, the aircraft in the master is looking good and these improvements will need a lot of work. I don't want to rush them. So IMO https://github.com/c172p-team/c172p-detailed/pull/1103 should stay opened.

Concerning issues, I think the only one we truly need to fix is https://github.com/c172p-team/c172p-detailed/pull/1102 all else is cosmetic.

> the image in the README.md is no longer visible. Could you make a new one with the new green retro livery perhaps?

I will create a new image for `README.md` but I will still hold back the splash screens. The modifications I am doing with wlbragg have the potential of affecting those, and creating good looking screenies does take time. But I will open a PR still tonight about fixing the `README.md` file.

**Comment by gilbertohasnofb** [comment]
@onox what do you think?:

![fg-0](https://user-images.githubusercontent.com/5700795/39959771-be4649b8-560e-11e8-8c54-b91b1bbea15b.jpg)

**Comment by wlbragg** [comment]
#1102 should be fixed now!

**Comment by onox** [comment]
The last two screenshots look nice with the clouds, but can you make one from the front instead of the back of the aircraft?

**Comment by onox** [comment]
> I will create a new image for README.md but I will still hold back the splash screens. The modifications I am doing with wlbragg have the potential of affecting those, and creating good looking screenies does take time. But I will open a PR still tonight about fixing the README.md file.

Ok, after PR #1102 has been merged, I'll make a release + branch tomorrow. That gives another day for testing and fixing any critical issues.

**Comment by gilbertohasnofb** [comment]
> The last two screenshots look nice with the clouds, but can you make one from the front(+ side a little bit) instead of the back of the aircraft?

Sure

**Comment by gilbertohasnofb** [comment]
@onox What about one of these?

![fg-0](https://user-images.githubusercontent.com/5700795/39961666-2d0aa050-5633-11e8-9f98-a1a5a4c81bda.jpg)
![fg-1](https://user-images.githubusercontent.com/5700795/39961667-2d2a20e2-5633-11e8-851b-8444a4e8247c.jpg)
![fg-2](https://user-images.githubusercontent.com/5700795/39961668-2d806074-5633-11e8-8f93-8d5eafd334ac.jpg)
![fg-3](https://user-images.githubusercontent.com/5700795/39961669-2dc964f4-5633-11e8-8b95-96a79c2bfae4.jpg)
![fg-4](https://user-images.githubusercontent.com/5700795/39961670-2e9ba5b8-5633-11e8-9ead-423c327a8bbd.jpg)

**Comment by onox** [comment]
I like 1-3. Maybe another one a bit more from the side and showing the new leading edge (and top side) of the wing a bit better?

**Comment by gilbertohasnofb** [comment]
Something like this?

![fgfs-screen-006](https://user-images.githubusercontent.com/5700795/39962144-771bf544-563e-11e8-81a9-ca43b8701614.png)
![fgfs-screen-007](https://user-images.githubusercontent.com/5700795/39962145-773906d4-563e-11e8-8a15-6bfb8b29ef98.png)
![fgfs-screen-009](https://user-images.githubusercontent.com/5700795/39962147-778e3d02-563e-11e8-8aa9-e07140bebb80.png)

![fgfs-screen-008](https://user-images.githubusercontent.com/5700795/39962146-776f8808-563e-11e8-81d1-ad649320fc6e.png)

**Comment by legoboyvdlp** [comment]
I like the second or third one, just fwiw.

**Comment by tonghuix** [comment]
![fgfs-screen-509](https://user-images.githubusercontent.com/423703/39962843-b92d7a9e-568d-11e8-85e0-44ad0c6bbcb1.png)
![fgfs-screen-507](https://user-images.githubusercontent.com/423703/39962844-b966cdc6-568d-11e8-9462-fea8a908b7c6.png)
![fgfs-screen-476](https://user-images.githubusercontent.com/423703/39962845-b998180e-568d-11e8-8f56-5e7dbecee7b7.png)

I took these screenshot yesterday, note that these are not original screenshots, I just make them a little bright.

**Comment by wlbragg** [comment]
2nd works.

**Comment by gilbertohasnofb** [comment]
@onox there is a release stopping bug reported https://github.com/c172p-team/c172p-detailed/issues/1106 and we must fix it urgently, please hold back if you were preparing a release today.

**Comment by onox** [comment]
@wlbragg @gilbertohasnofb I've shorten the the release notes a bit here in preparation for updating `NEWS.md`.

**Comment by legoboyvdlp** [comment]
Awesome!

**Comment by gilbertohasnofb** [comment]
Thanks to you all, in particular for saint @wlbragg and all his patience :rofl: 

:tada: :tada: :tada:

**Comment by stuartbuchanan** [comment]
Great.  I'm away this so it will be Monday before I get the chance to check
it in.

**Comment by wkitty42** [comment]
On 05/15/2018 07:40 AM, stuartbuchanan wrote:
> Great. I'm away this so it will be Monday before I get the chance to check it
> in.

ummm... the FG 2018.2 release branching is in three days, isn't it... i think 
the team is aiming to get this into the 2018.2.1 release... or am i missing 
something?

just asking...

**Comment by gilbertohasnofb** [comment]
> ummm... the FG 2018.2 release branching is in three days, isn't it... i think the team is aiming to get this into the 2018.2.1 release... or am i missing something?

Yes, we intended to get this release merged before the 2018.2.1 release. @stuartbuchanan is there anyone else who could take care of merging this aircraft to FGData in time?

**Comment by stuartbuchanan** [comment]
Sorry - hadn't realized that timing wouldn't work.


I'd suggest trying Torsten, Richard or James.

-Stuart

**Comment by gilbertohasnofb** [comment]
@stuartbuchanan Thanks for the info, I just wrote the dev list.

**Comment by gilbertohasnofb** [comment]
Thanks @onox for some reason my original email didn't go through, I resent it now and hopefully it will arrive this time.

**Comment by gilbertohasnofb** [comment]
@onox are you following the discussion on the list about how to tackle these merges of our repo into FGData? I am not very knowledgeable about these things so it would be ideal if you can keep an eye on the discussion over there

**Comment by onox** [comment]
How they want to automate it is not really my concern. I just try to create a release branch in time with a predictable name (`release/2018.1`, `release/2018.2`, etc.)

Like @zakalawe said, removing the data requires another download, so doing a git checkout of the release branch and then rsyncing it might be easier.
